src/readableview.js

```diff
@@ -0,0 +1,67 @@
+/** @flow */
```

Comment:
I know the GSG is against it, but we *are* currently using CamelCase in file names in CycleDash, and it makes even more sense, I'd think, when using ES6-style classes like we're doing here and exporting only them from the module. I think GSG is a bit out of date when it comes to e.g. flow, ES6, and React.js. If you still to use all lower-case, then my preference is with `readable-view.js`.